client/wfm/src/result/StepResult.ts

```diff
@@ -0,0 +1,35 @@
+import { STATUS } from '../status';
+import { Step } from '../step/Step';
+
+/**
+ * Output of a single {@link Step} in the context of a {@link WorkOrder}
+ */
+export interface StepResult {
+  /**
+   * Form data output of the {@link Step}'s execution
+   */
+  submission?: {
```

Comment:
Not sure what you mean by this.
```
submission: {
    "fuel" : 25,
    "tires" : true,
    "lights" : true
}
```
Is sample data from vehicle inspection, currently the form model being passed from angular.
What do you think it should shape like? Repeated keys should be invalid I guess.

I think this should be in a new issue.